api/src/test/java/com/epam/pipeline/acl/plugin/PluginAssignmentServiceTest.java

```diff
@@ -0,0 +1,269 @@
+package com.epam.pipeline.acl.plugin;
+
+import com.epam.pipeline.dao.plugin.UIPluginAssignmentRepository;
+import com.epam.pipeline.dao.plugin.UIPluginRepository;
+import com.epam.pipeline.dto.plugin.UIPlugin;
+import com.epam.pipeline.dto.plugin.UIPluginAssignment;
+import com.epam.pipeline.entity.plugin.UIPluginAssignmentEntity;
+import com.epam.pipeline.entity.plugin.UIPluginEntity;
+import com.epam.pipeline.exception.ObjectNotFoundException;
+import com.epam.pipeline.mapper.plugin.UIPluginAssignmentMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class PluginAssignmentServiceTest {
+
+    private static final Long PLUGIN_ID = 1L;
+    private static final Long ASSIGNMENT_ID = 1L;
+    private static final Long TOOL_ID = 2L;
+    private static final Long PIPELINE_ID = 3L;
+    private static final String VERSION = "v1.0";
+
+    @Mock
+    private UIPluginAssignmentRepository assignmentRepository;
+
+    @Mock
+    private UIPluginRepository pluginRepository;
+
+    @Mock
+    private UIPluginAssignmentMapper mapper;
+
+    @InjectMocks
+    private PluginAssignmentService pluginAssignmentService;
+
+    private UIPlugin pluginDto;
+    private UIPluginEntity pluginEntity;
+    private UIPluginAssignment assignmentDto;
+    private UIPluginAssignmentEntity assignmentEntity;
+
+    @Before
+    public void setUp() {
+        pluginDto = new UIPlugin();
+        pluginDto.setId(PLUGIN_ID);
+
+        pluginEntity = new UIPluginEntity();
+        pluginEntity.setId(PLUGIN_ID);
+
+        assignmentDto = new UIPluginAssignment();
+        assignmentDto.setId(ASSIGNMENT_ID);
+        assignmentDto.setPlugin(pluginDto);
+        assignmentDto.setToolId(TOOL_ID);
+        assignmentDto.setVersion(VERSION);
+
+        assignmentEntity = new UIPluginAssignmentEntity();
+        assignmentEntity.setId(ASSIGNMENT_ID);
+        assignmentEntity.setPlugin(pluginEntity);
+        assignmentEntity.setToolId(TOOL_ID);
+        assignmentEntity.setVersion(VERSION);
+    }
+
+    @Test
+    public void testGetAssignmentsByToolIdAndVersion() {
```

Comment:
Could we use `should` notation for test names? `shouldGetAssignmentsByToolIdAndVersion`